content/tutorials/quickstart/mint-and-burn-nftokens.md

```diff
@@ -122,15 +114,15 @@ Connect to the ledger and get the account wallets.
 ```
 
 
-          results = 'Connecting to ' + getNet() + '....'
-          document.getElementById('standbyResultField').value = results
-          let net = getNet()
-          const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
-          const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
-          const client = new xrpl.Client(net)
-        await client.connect()
-          results += '\nConnected. Minting NFToken.'
-          document.getElementById('standbyResultField').value = results
+  results = 'Connecting to ' + getNet() + '....'
+  document.getElementById('standbyResultField').value = results
+  let net = getNet()
+  const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  const client = new xrpl.Client(net)
```

Comment:
The blurb on line 111 above says "account wallets" but this code only gets one wallet now. Not sure which one should be fixed.